build.gradle

```diff
@@ -111,6 +113,7 @@ compileScala {
 }
 
 dependencies {
+    compile files("libs/genomicsdb-0.6.4.jar")
```

Comment:
We should be able to change this to:

```scala
compile group: 'com.intel', name: 'genomicsdb', version: '0.6.4-proto-3.0.0-beta-1'
```

now that it includes the dylib files we need. Am I forgetting a different issue we had with this?